src/utils/FalsyValuesUtils.ts

```diff
@@ -5,8 +5,8 @@ export const convertUndefinedAndNullToEmptyString = (value: any) =>
         ? ''
         : value;
 
-export const callIfDefined = (callback: any, ...args: any[]) => {
+export const callIfDefined = (callback: any, ...args: any[]): any => {

```

Comment:
Use a template as return Type could be nice!

Reply:
Not sure how TBH, do you have a snippet?

Reply:
`export const callIfDefined = <T>(callback: () => T, ...args: any[]): T => {
    if (_.isFunction(callback)) {
        callback(...args);
        return callback(...args);
     }
};`